src/repo/stat.js

```diff
@@ -41,5 +41,11 @@ module.exports = (createCommon, options) => {
         expectIsRepo(null, res)
       })
     })
+
+    it('should get human readable repo stats', async () => {
+      const res = await ipfs.repo.stat({ human: true })
+
+      expectIsRepoHumanized(null, res)
```

Comment:
Please move these assertions into this test. The function is not being reused so there's no need for it to be extracted.